src/components/Datepicker.vue

```diff
@@ -89,45 +89,27 @@ export default {
         return val === null || val instanceof Date || typeof val === 'string'
       }
     },
-    name: {
-      value: String
-    },
-    id: {
-      value: String
-    },
+    name: String,
```

Comment:
@charliekassel you can see example in vue documentation here https://vuejs.org/v2/guide/components.html#Prop-Validation. In that example they have used both type of property assignment short and long.